src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -19,24 +19,75 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The miners building.
+ */
 public class BuildingMiner extends AbstractBuildingWorker
 {
+    /**
+     * The NBT Tag to store the floorBlock
+     */
     private static final    String              TAG_FLOOR_BLOCK         = "floorBlock";
+    /**
+     * The NBT Tag to store the fenceBlock
+     */
     private static final    String              TAG_FENCE_BLOCK         = "fenceBlock";
+    /**
+     * The NBT Tag to store the starting level of the shaft.
+     */
     private static final    String              TAG_STARTING_LEVEL      = "startingLevelShaft";
+    /**
+     * The NBT Tag to store list of levels.
+     */
     private static final    String              TAG_LEVELS              = "levels";
+    /**
+     * The NBT Tag to store if the shaft has been cleared.
+     */
     private static final    String              TAG_CLEARED             = "clearedShaft";
+    /**
+     * The NBT Tag to store the location of the shaft.
+     */
     private static final    String              TAG_SLOCATION           = "shaftLocation";
+    /**
+     * The NBT Tag to store the vector-x of the shaft
+     */
     private static final    String              TAG_VECTORX             = "vectorx";
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)